misc/test

```diff
@@ -7,6 +7,8 @@ CURR=$(pwd)
 TARGETDIR=".${CURR#"$DIR"}"
 cd "$DIR"
 
+source "$DIR/misc/test-utils.sh"
```

Comment:
This script should probably be in the `scripts` sub-directory instead. We're generally trying to reduce the amount of files immediately visible here, to varying degrees of success :P